meta-luneos/recipes-support/pidgin/pidgin-sipe_1.21.1.bb

```diff
@@ -0,0 +1,29 @@
+SUMMARY = "Protocol plugin for Office 365/Lync/OCS for Adium, Pidgin, Miranda and Telepathy IM Framework"
+SECTION = "webos/services"
+LICENSE = "GPLv2.0"
+LIC_FILES_CHKSUM = "file://COPYING;md5=751419260aa954499f7abaabaa882bbe"
+
+DEPENDS = "pidgin intltool-native"
+
+inherit pkgconfig
+inherit autotools
+
+SRC_URI = "${SOURCEFORGE_MIRROR}/sipe/pidgin-sipe-${PV}.tar.xz"
+SRC_URI[md5sum] = "96ea396adb5c8a1bb1749d778892605c"
+SRC_URI[sha256sum] = "651af55d65cbdf5bdbee0366bd5151bec4152c910743aa432204d015893fe444"
+
+PACKAGECONFIG ??= "nss krb5"
+PACKAGECONFIG[nss] = "--enable-nss=yes,--enable-nss=no,nss"
+PACKAGECONFIG[openssl] = "--enable-openssl=yes,--enable-openssl=no,openssl"
+PACKAGECONFIG[krb5] = "--with-krb5=yes,--with-krb5=no,krb5"
+#PACKAGECONFIG[voice_and_video] = "--with-vv=yes,--with-vv=no,libnice gstreamer"
+PACKAGECONFIG[telepathy] = "--enable-telepathy=yes,--enable-telepathy=no,telepathy-glib gmime"
+#PACKAGECONFIG[gssapi_only] = "--enable-gssapi-only=yes,--enable-gssapi-only=no,krb5"
+PACKAGECONFIG[debug] = "--enable-debug=yes,--enable-debug=no,valgrind"
+
+FILES_${PN} += " \
+    ${libdir} \
+    ${datadir}/appdata \
+    ${datadir}/pixmaps \
+"
```

Comment:
libsipe.so = elf library. 

I've updated based on your comments: https://github.com/webOS-ports/meta-webos-ports/pull/186/commits/6eb6543ee13abaab1be0d9d97a5a79d14dd32f31

Also added freeimage recipe from upstream repo required for whatsapp purple https://github.com/openembedded/openembedded/blob/fabd8e6d07d3cd0cc93c2a0fc804f8c8f316c649/recipes/ogre/freeimage_3.11.0.bb